requirements.txt

```diff
@@ -24,6 +24,40 @@ rsa==3.4.2
 urllib3==1.24.3
 more-itertools==4.3.0
 typing-extensions==3.7.4
-werkzeug==0.15.5
+Werkzeug==0.15.5
```

Comment:
Why the case change?

Reply:
Note the name on the url and the package name on PyPI
https://pypi.org/project/werkzeug/